accessReviewScheduleDefinition docs refresh (#12447)
microsoftgraph/microsoft-graph-docs: concepts/accessreviews-reviewers-concept.md

---

> title: "Assign reviewers to your access review definition"

Reviewer: The title field should match the H1 title. You'll want to make sure that the meta title includes all the important keywords because it's used for search.

description: "Learn how to use the access reviews API in Microsoft Graph to assign access reviewers."
author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# Assign reviewers to your access review definition using the Microsoft Graph API

> The Azure AD [access reviews API](/graph/api/resources/accessreviewsv2-root?view=graph-rest-beta&preserve-view=true) allow you to programmatically review access—of all users, a set of users (for example, guest users only), as well as service principals and groups—to your Azure AD resources.

Reviewer: This sentence still is not clear. Is it trying to say that the APIs "allows you to programmatically review the access that all users, a set of users, and service principals and groups have to your Azure AD resources." ?

> [!NOTE]

> > The [access reviews API](/graph/api/resources/accessreviewsv2-root?view=graph-rest-beta&preserve-view=true) are currently available in only the Microsoft Graph beta endpoint. Do not use them in production apps, as they are subject to change without notice.

Reviewer: Change "are" to "is" because we made API singular.

> The primary reviewers are configured in the **reviewers** property of the access reviews [accessReviewScheduleDefinition](/graph/api/resources/accessreviewscheduledefinition?view=graph-rest-beta&preserve-view=true) resource.  In addition, you can specify fallback reviewers using **fallbackReviewers** property. These properties are not required when you create a self-review (where users review their own access).

Reviewer: To help readability for global readers, please change "using" to "by using". This removes potential ambiguity.

## Configure reviewers

> To configure the reviewers and fallback reviewers, set the values of **query**, **queryRoot**, and **queryType** properties of **accessReviewReviewerScope**. [See the accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true) resource for descriptions of these properties.

Reviewer: Global: For cross-references, always tell them what the information is before providing the link; otherwise they potentially click away without getting the full context. So it should be "For descriptions of these properties, see [accessReviewReviewerScope]." Also note that the link text should just be the resource name/title that you're linking to, not the verb.

### Example 1: A specific user as the reviewer

```http
"reviewers": [
    {
        "query": "/users/{user id}",
        "queryType": "MicrosoftGraph"
    }
]
```

### Example 2: Members of a group as reviewers

```http
"reviewers": [
    {
        "query": "/groups/{group id}}/transitiveMembers",
        "queryType": "MicrosoftGraph"
    }
]
```

### Example 3: Group owners as reviewers
```http
"reviewers": [
    {
        "query": "./owners",
        "queryType": "MicrosoftGraph"
    }
]
```

> To specify only a specific non-guest groups owners from a specific country as the reviewers, specify as follows:

Reviewer: This sentence isn't quite clear. You can remove "specify as follows" because you begin the sentence with "To specify". The phrase "non-guest groups owners" is hard to parse. Maybe you mean "group owners"?

---
title: "Assign reviewers to your access review definition"
description: "Use the reviewers property of accessReviewsScheduleDefinition to assign reviewers of your access review."
author: "isabelleatmsft"

> localization_priority: Normal

Reviewer 2: Missing ms.prod and doc_type.

> [!NOTE]
> The [access reviews APIs](/graph/api/resources/accessreviewsv2-root?view=graph-rest-beta&preserve-view=true) are currently available in only the Microsoft Graph beta endpoint. Do not use them in production apps, as they are subject to change without notice.

The primary reviewers are configured in the **reviewers** property of the access reviews [accessReviewScheduleDefinition](/graph/api/resources/accessreviewscheduledefinition?view=graph-rest-beta&preserve-view=true) resource, a property of the type [accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true).  In addition, you can specificy fallback reviewers using **backupReviewers** or **fallbackReviewers**, another property of the type [accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true)

> Note that this property is not required when creating a self-review (where users review their own access).

Reviewer 2: ```suggestion
This property is not required when a self-review (where users review their own access) is created.
```

> ### Use accessReviewReviewerScope to configure reviewers

Reviewer 2: ```suggestion
### Configure reviewers
```

> ### Use accessReviewReviewerScope to configure reviewers

Reviewer 2: Should be an H2.

> To configure the reviewers and/or fallback reviewers, set the values of **query**, **queryRoot**, and **queryType** properties of accessReviewReviewerScope. [See the accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true) resource for descriptions of these properties.

Reviewer 2: ```suggestion
To configure the reviewers and fallback reviewers, set the values of **query**, **queryRoot**, and **queryType** properties of an [accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true) resource.
```

> 

Reviewer 2: Missing an ## Examples heading

> Example configurations of the **reviewers** or **backupReviewers** property using the accessReviewReviewerScope include the following:

Reviewer 2: ```suggestion
Example configurations of the **reviewers** or **backupReviewers** property using the accessReviewReviewerScope.
```

> #### Example 1: A specific user as the reviewer

Reviewer 2: Should be an H3.

> #### Example 1: A specific user as the reviewer

Reviewer 2: Should be an H3.

```http
"reviewers": [
    {
        "query": "/users/{user id}",
        "queryType": "MicrosoftGraph"
    }
]
```

> #### Example 2: Members of a group as reviewers

Reviewer 2: Should be an H3.

```http
"reviewers": [
    {
        "query": "/groups/{group id}}/transitiveMembers",
        "queryType": "MicrosoftGraph"
    }
]
```

> #### Example 3: Group owners as reviewers

Reviewer 2: Should be an H3.

---

> title: "Assign reviewers to your access review definition"

Reviewer: Do you want these two new articles to be included in the concept TOC?

Author: No. They are not tutorials, just how-to like articles for the configuration of the 2 properties.

Reviewer: How will users discover them if they aren't in the TOC? Generally we do expose How to articles in the TOC.

Author: We've added links from the property descriptions and the 3 complex resources we're configuring. For example, the descriptions of **scope** and **reviewers** properties of **accessReviewsScheduleDefinition** have links to the scope and reviewers configuration articles respectively.

> description: "Use the reviewers property of accessReviewsScheduleDefinition to assign reviewers of your access review."

Reviewer: It might be better to make this description more high level - "Learn how to use the access reviews API in Microsoft Graph to assign access reviewers."

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

> # Assign reviewers to your access review definition

Reviewer: The title and H1 should include important keywords for SEO, so you might want to add "using the Microsoft Graph access review API".

> Azure AD [access reviews APIs](/graph/api/resources/accessreviewsv2-root?view=graph-rest-beta&preserve-view=true) allow you to programmatically review access to your Azure AD resources. This can be all users, a set of users (for example, guest users only), as well as service principals and groups with access to Azure AD resources.

Reviewer: We can refer to it as an API, so revise as: "The Azure AD access reviews API allows you to...". I'm not clear about the meaning of the second sentence. What does "This" refer to? Azure AD resources? Or the types of users who can be assigned as reviewers? In the first sentence, is "programmatically review access" what you intend to say or is it "programmatically assign access reviewers" as the title implies?

Author: This sentence is intended as a HL overview of the access reviews API. The intent of the 1st sentence is ""programmatically review access". I've reworded the sentence, kindly review.

_The Azure AD access reviews API allow you to programmatically review access—of all users, a set of users (for example, guest users only), as well as service principals and groups—to your Azure AD resources._

> [!NOTE]

> > The [access reviews APIs](/graph/api/resources/accessreviewsv2-root?view=graph-rest-beta&preserve-view=true) are currently available in only the Microsoft Graph beta endpoint. Do not use them in production apps, as they are subject to change without notice.

Reviewer: Revise as "access reviews API is".

> The primary reviewers are configured in the **reviewers** property of the access reviews [accessReviewScheduleDefinition](/graph/api/resources/accessreviewscheduledefinition?view=graph-rest-beta&preserve-view=true) resource, a property of the type [accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true).  In addition, you can specify fallback reviewers using **fallbackReviewers**, another property of the type [accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true).

Reviewer: Is it important to call out the type for the two properties? The construction in both sentences is awkward.

Author: Reworded, since I reference the type in the H2 section that follows.

> This property is not required when a self-review (where users review their own access) is created.

Reviewer: Which property are you referring to here - reviewers, or fallbackReviewers?

Author: Both, reworded.

## Configure reviewers

To configure the reviewers and fallback reviewers, set the values of **query**, **queryRoot**, and **queryType** properties of an accessReviewReviewerScope. [See the accessReviewReviewerScope](/graph/api/resources/accessreviewreviewerscope?view=graph-rest-beta&preserve-view=true) resource for descriptions of these properties.

> Example configurations of the **reviewers** or **fallbackReviewers** property using the accessReviewReviewerScope include the following:

Reviewer: We don't introduce headings with a colon. I would remove this sentence because the Examples heading already makes that clear.

> ### Examples

Reviewer: You can streamline this by removing this H3 and making each example an H3.

#### Example 1: A specific user as the reviewer

```http
"reviewers": [
    {
        "query": "/users/{user id}",
        "queryType": "MicrosoftGraph"
    }
]
```

#### Example 2: Members of a group as reviewers

```http
"reviewers": [
    {
        "query": "/groups/{group id}}/transitiveMembers",
        "queryType": "MicrosoftGraph"
    }
]
```

#### Example 3: Group owners as reviewers
```http
"reviewers": [
    {
        "query": "./owners",
        "queryType": "MicrosoftGraph"
    }
]
```

To specify only a specific non-guest groups owners from a specific country as the reviewers, specify as follows:

```http
"reviewers": [
    {
        "query": "/groups/{group id}/owners?$filter=microsoft.graph.user/userType eq 'Member' and microsoft.graph.user/country eq 'USA'",
        "type": "MicrosoftGraph”
    }
]
```

> #### Example 4: People managers as the reviewers

Reviewer: Remove "the" here.

```http
"reviewers": [
    {
        "query": "./manager",
        "queryType": "MicrosoftGraph",
        "queryRoot": "decisions"
    }
]
```
Because `./manager` is a relative query, specify the **queryRoot** property with the value `decisions`.

> 

Reviewer: Do you need to provide response examples?

Author: Not in this article which focuses only on the configuration of this property (though the request & response are similar). In the POST API doc, we've added a full example that uses this configuration.
